hackathon_site/dashboard/frontend/src/components/orders/OrderCard/OrderCard.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
nice job with this test! Could you wrap it in a describe block in case we need to write more tests.

Also, you'll need tests for the Orders page, testing that all the order cards in mockPendingOrders are displaying properly (based on the order status)